csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -31,8 +31,8 @@
   <!--TODO: this works for single platform only. Need separate packaging scripts for multi-target packaging -->
   <!--TODO: Find a way to bundle the native symbol files properly -->
   <ItemGroup>
-    <None Include="$(OnnxRuntimeCsharpRoot)\..\include\**\*.*"
-          PackagePath="\build\native\include"
+    <None Include="$(OnnxRuntimeCsharpRoot)\..\include\onnxruntime\core\session\*"
```

Comment:
I have asked Ryan moving the C API headers to a different folder. 
Because the source code in core/framework depends on these header files, which is in core/session. It looks like a circular dependency.

Reply:
better copy by the exact header name. Only one or 2 headers should be copied.

Reply:
We'll need to update this location when the location changes.  Currently, there are only 2 files in this directory -- so only 2 headers are copied.